jumpscale/threesdk/threebot.py

```diff
@@ -1,59 +1,20 @@
-import requests
 import os
 
-from .container import Container
-from . import settings
 
-from jumpscale.core.exceptions import Value
-from jumpscale.data.encryption import mnemonic
-from jumpscale.data.encryption.exceptions import FailedChecksumError
-from jumpscale.data.nacl.jsnacl import NACL
-from jumpscale.tools.console import ask_string, ask_choice, printcolors, ask_yes_no
 from jumpscale.core.config import get_current_version
+from jumpscale.core.exceptions import Value
+
+from . import settings
+from .container import Container
+from .identitymanager import IdentityManager
 
 DEFAULT_CONTAINER_NAME = "3bot-ng"
 DEFAULT_IMAGE = "threefoldtech/js-ng"
 PERSISTENT_STORE = os.path.expanduser("~/.config/jumpscale/containers")
 
-
 NETWORKS = {"mainnet": "explorer.grid.tf", "testnet": "explorer.testnet.grid.tf", "devnet": "explorer.devnet.grid.tf"}
```

Comment:
this variable is not used here anymore

Reply:
we use it actually, we ask user to provide network name, it's the map. no ?